<launch>
	
	<!-- which topological map to simulate. if undefined one will be created -->
  	<arg name="map" default="" />

	<!-- Or the path to a yaml topological map to load and simulate -->
  	<arg name="yaml_map" default="" />

  	<!-- whether to simulate travel time or not -->
  	<arg name="simulate_time" default="false" />

	<node pkg="topological_utils" type="dummy_topological_navigation.py" name="dummy_topological_navigation" output="screen">
		<!-- if simulate time is true then this node uses the estimator service to get the travel time, then sleeps for that long -->
		<param name="simulate_time" value="$(arg simulate_time)"/>
		<param name="map" value="$(arg map)" />
		<param name="yaml_map" value="$(arg yaml_map)" />
	</node>

	<node pkg="topological_navigation" type="travel_time_estimator.py" name="travel_time_estimator" output="screen"/>

	<node pkg="topological_navigation" type="topological_prediction.py" name="topological_prediction"/> 

	<!-- <node pkg="topological_navigation" type="speed_based_prediction.py" name="topological_prediction"/>  -->


	<!--node pkg="fremenserver" type="fremenserver" name="fremenserver"/--> 


</launch>
